src/python_testing/TC_IDM_1_4.py

```diff
@@ -0,0 +1,167 @@
+#
+#    Copyright (c) 2023 Project CHIP Authors
+#    All rights reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+from dataclasses import dataclass
+
+import chip.clusters as Clusters
+from chip import ChipUtility
+from chip.exceptions import ChipStackError
+from chip.interaction_model import InteractionModelError, Status
+from matter_testing_support import MatterBaseTest, async_test_body, default_matter_test_main, type_matches
+from mobly import asserts
+
+
+@dataclass
+class FakeInvalidBasicInformationCommand(Clusters.BasicInformation.Commands.MfgSpecificPing):
+    @ChipUtility.classproperty
+    def must_use_timed_invoke(cls) -> bool:
+        return False
+
+
+class TC_IDM_1_4(MatterBaseTest):
+
+    @async_test_body
+    async def test_TC_IDM_1_4(self):
+        dev_ctrl = self.default_controller
+        dut_node_id = self.dut_node_id
+
+        self.print_step(0, "Commissioning - already done")
+
+        self.print_step(1, "Get remote node's MaxPathsPerInvoke")
+        session_parameters = dev_ctrl.GetRemoteSessionParameters(dut_node_id)
+        max_paths_per_invoke = session_parameters.maxPathsPerInvoke
+
+        asserts.assert_greater_equal(max_paths_per_invoke, 1, "Unexpected error returned from unsupported endpoint")
+        asserts.assert_less_equal(max_paths_per_invoke, 65535, "Unexpected error returned from unsupported endpoint")
+
+        self.print_step(2, "Sending `MaxPathsPerInvoke + 1` InvokeRequest if it fits into single MTU")
```

Comment:
This doesn't quite match the test plan. Probably should update the test plan. Not a super high priority though.

Reply:
There is a note in the testplan about this:
https://github.com/CHIP-Specifications/chip-test-plans/blob/master/src/interactiondatamodel.adoc?plain=1#L460

Do you mean that we should add this to the teststep itself?

Reply:
ah, yeah, it's the note that was throwing me off. If you can add it to the test step directly, that would be good.